src/CMakeLists.txt

```diff
@@ -2,11 +2,21 @@ project(OpenCamLib)
 
 cmake_minimum_required(VERSION 2.4)
 #set( CMAKE_SOURCE_DIR ${CMAKE_SOURCE_DIR} )
+set(CMAKE_VERBOSE_MAKEFILE ON)
 
 if(COMMAND cmake_policy)
   cmake_policy(SET CMP0003 NEW)
+  cmake_policy(SET CMP0025 NEW)
 endif(COMMAND cmake_policy)
 
+if (CMAKE_VERSION VERSION_LESS "3.1")
+  if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11")
+  endif ()
+else ()
+  set (CMAKE_CXX_STANDARD 11)
+endif ()
+
 # some options to set what is built:
```

Comment:
The above changes are needed to tell cmake to use the right v11 compiler, I found / modified it from the following stack overflow answer: https://stackoverflow.com/questions/10851247/how-to-activate-c-11-in-cmake

Right now there is only a GNU compiler check, we should probably add Clang and others as well, or perhaps use another way to force V11.